src/player.cpp

```diff
@@ -12113,9 +12113,9 @@ std::string player::is_snuggling() const
     }
 
     for( auto candidate = begin; candidate != end; ++candidate ) {
-        if( !candidate->is_armor() ) {
+        if( !candidate->is_armor() || candidate->is_gun() ) {
```

Comment:
I suppose not, but it's on you if someone makes a gun that has warmth! Actually I bet in that case a survivor would snuggle it...